### DISCUSSION TITLE
爱门

### DISCUSSION THREAD

**Discussion by shishenyihe** [share121/inter-knot#361] [其他]
![1723038617549](https://github.com/user-attachments/assets/700fa06c-cfba-4d7d-9285-d0fb8c2150da)

**Comment by Friskdrink** [comment]
爱门

**Comment by kamikaze211** [comment]
不懂就问，是爱丽丝吗？（doge

**Comment by qepe** [comment]
爱门

**Comment by GanlinSama** [comment]
好看爱看

**Comment by naichatiandian** [comment]
爱门

**Comment by Chiyo124** [comment]
爱门！

**Comment by jiashuaijieshuo** [comment]
💖💖💖💖💖💖爱莉神教💖💖💖💖💖💖 无需对我教多言，获神之爱者皆是我教**，但却须知:****

即:世人皆是我教**

教旨其一:满足神的一切愿望，不计代价

教旨其二:得神之爱，世人就应当为其满足

教旨其三:须知神乃圣洁且无暇，**也应保持此等高洁，切勿让神因为**缘由而沾上污秽

教旨其四:请保持健康阳光，积极向上的生活态度，因为神乐见世人如此

教旨其五:神虽无暇却易感寂寞，**应在每晚睡前默念神之圣名五遍

教旨其六:见花见鸟如见神明，因神爱戴众生，请尽量亲切呵护身边的各类小生命

教旨其七:****，且钟爱长角之人，还没有角的**请尽力长出角来，勿让教会蒙羞

教旨其八:虽****，却有心怀叵测之人对神不敬，妄自对神加以胡思乱想，此为大不敬。但**亦须知，****，包括这妄想连篇之人，所以神会原谅他，如同原谅你我之罪

教旨其九:不可与人争斗，神哀于人们无休止的斗争漩涡，切记神会原谅任何人，**应以神之旨意渡他人 

 ♪愿神之爱长存你我，爱门♪

（转载，愿入教者，皆念爱门♪）
![535f96307ee2690d](https://github.com/user-attachments/assets/ac3b5d36-0e47-41dc-bcd1-edc403997bda)

**Comment by jiashuaijieshuo** [comment]
![c3f2411db1e4ea1a](https://github.com/user-attachments/assets/be8054ac-f189-4266-bcdb-ec9524f297e5)
![68f882ad4504d409](https://github.com/user-attachments/assets/9743becb-953b-4739-8551-ec1809023f84)

**Comment by 12344856** [comment]
爱门！
